Contents/mods/!pz_debugTools/media/lua/client/Util_Functions_Community_client.lua

```diff
@@ -0,0 +1,25 @@
+--local util = require "Util_Functions_Community_client.lua"
+local util = {}
+
+function util.markAllStasBuildingshOnMap(show,clear)
```

Comment:
Stas**h** ;)

Reply:
heh, that h is at wrong place. fixed typo